lib/Tracing.js

```diff
@@ -73,15 +72,21 @@ class Tracing {
    */
   async _readStream(handle, path) {
     let eof = false;
-    const file = await openAsync(path, 'w');
+    let file;
+    if (path)
+      file = await openAsync(path, 'w');
+    const bufs = [];
     while (!eof) {
       const response = await this._client.send('IO.read', {handle});
       eof = response.eof;
+      bufs.push(new Buffer(response.data));
       if (path)
         await writeAsync(file, response.data);
     }
-    await closeAsync(file);
+    if (path)
+      await closeAsync(file);
     await this._client.send('IO.close', {handle});
+    return Buffer.concat(bufs);
```

Comment:
@yanivefraim I'm recalling now that back in the days we didn't implement buffers here because buffers are constrained to 1Gb in size.

It's unlikely someone has such a huge trace, but it's nice to know what will happen in this case. 
If this line throws an error, let's catch it and return `null` to not compromise existing users.

Reply:
@aslushnikov - sure, done. 10x!

Reply:
Just to confirm, I tested it and a large buffer throws `Uncaught RangeError: Invalid typed array length`.